testing/cli/test_cli_sct_process_segmentation.py

```diff
@@ -96,6 +97,38 @@ def test_sct_process_segmentation_check_normalize_missing_predictor(dummy_3d_mas
         assert e.value.code == 2
 
 
+def test_sct_process_segmentation_check_normalize_PAM50(tmp_path):
+    """ Run sct_process_segmentation with -normalize PAM50"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    sct_process_segmentation.main(argv=['-i', sct_test_path('t2', 't2_seg-manual.nii.gz'), '-normalize-PAM50', '1',
+                                        '-perslice', '1', '-vertfile', sct_test_path('t2', 't2_seg-manual_labeled.nii.gz'), '-o', filename])
+    with open(filename, "r") as csvfile:
+        reader = csv.DictReader(csvfile, delimiter=',')
+        rows = list(reader)
+        row = rows[827]
+        assert row['Slice (I->S)'] == '827'
+        assert float(row['MEAN(area)']) == pytest.approx(71.96880493869594)
+        assert row['VertLevel'] == '5'
+
+
+def test_sct_process_segmentation_check_normalize_PAM50_missing_perslice(tmp_path):
+    """ Run sct_process_segmentation with -normalize PAM50 when missing perslice argument"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    with pytest.raises(SystemExit) as e:
+        sct_process_segmentation.main(argv=['-i', sct_test_path('t2', 't2_seg-manual.nii.gz'), '-normalize-PAM50', '1',
+                                            '-vertfile', sct_test_path('t2', 't2_seg-manual_labeled.nii.gz'), '-o', filename])
+        assert e.value.code == 2
+
+
+def test_sct_process_segmentation_check_normalize_PAM50_missing_vertfile(tmp_path):
+    """ Run sct_process_segmentation with -normalize PAM50 when missing -vertfile"""
+    filename = str(tmp_path / 'tmp_file_out.csv')
+    with pytest.raises(SystemExit) as e:
+        sct_process_segmentation.main(argv=['-i', sct_test_path('t2', 't2_seg-manual.nii.gz'), '-normalize-PAM50', '1',
+                                            '-perslice', '1', '-o', filename])
+        assert e.value.code == 2
```

Comment:
Since the "missing vertfile" error was changed to a `FileNotFoundError`, we need to switch the `pytest.raises` statement, as the test now fails to catch the error:

```
_____ test_sct_process_segmentation_check_normalize_PAM50_missing_vertfile _____
Traceback (most recent call last):
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/testing/cli/test_cli_sct_process_segmentation.py", line 127, in test_sct_process_segmentation_check_normalize_PAM50_missing_vertfile
    sct_process_segmentation.main(argv=['-i', sct_test_path('t2', 't2_seg-manual.nii.gz'), '-normalize-PAM50', '1',
  File "/home/runner/work/spinalcordtoolbox/spinalcordtoolbox/spinalcordtoolbox/scripts/sct_process_segmentation.py", line 394, in main
    raise FileNotFoundError("The vertebral level file must exist to use -normalize-PAM50.")
FileNotFoundError: The vertebral level file must exist to use -normalize-PAM50.
```

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3977/commits/55699d2571a351a354c4ca662704e8474ceeb08b